### PR TITLE
Added some Test Cases 

### PR DESCRIPTION
**Pull request by MANN3005**
new tests: workflow, java, android, javascript

### CODE REVIEW

**Comment by MANN3005** [comment]
I have added this test cases to cover some edge case which were not being checked for smoother functioning.

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by MANN3005** [comment]
How can I add my name/email to the commit ?

**Comment by MANN3005** [comment]
If still any fix is needed please tell me I will do it.

**Comment by wing328** [comment]
https://github.com/OpenAPITools/openapi-generator/actions/runs/6764680408/job/18426301169?pr=16994 failed

File: samples/client/echo_api/java/okhttp-gson/src/test/java/org/openapitools/client/model/BirdAndCategoryTest.java [hunk 1]
```diff
@@ -34,15 +34,19 @@ public class BirdAndCategoryTest {
      */
     @Test
     public void testBirdAndCategory() {
-        // TODO: test BirdAndCategory
+        birdAndCategory.setSize("Small");
+        assertEquals("Small", birdAndCategory.getSize());
+
+        birdAndCategory.setSize("Large");
```
Comment (wing328) [inline samples/client/echo_api/java/okhttp-gson/src/test/java/org/openapitools/client/model/BirdAndCategoryTest.java:40]: should be `model` instead of `birdAndCateogry` ? 

same for line 37

Comment (MANN3005) [reply samples/client/echo_api/java/okhttp-gson/src/test/java/org/openapitools/client/model/BirdAndCategoryTest.java:40]: Yeah let me change and push it.

**Comment by MANN3005** [comment]
Have Pushed the required changes.

**Comment by wing328** [comment]
a different error: https://github.com/OpenAPITools/openapi-generator/actions/runs/6779625993/job/18428069487?pr=16994

can you please run the the test locally to confirm ?

**Comment by MANN3005** [comment]
Yes will do before next push, sorry about that.
